### DISCUSSION TITLE
how to disable ctrl+c in pudb

### DISCUSSION THREAD

**Discussion by guoyejun** [inducer/pudb#597] [General]
Sometimes, I wrongly press ctrl+c when I actually want to press ctrl+x, and the program exits.

I have tried with "trap '' INT" in the terminal, but it does not work.

thanks.

**Comment by asmeurer** [comment]
This should just work out of the box. What terminal do you use? Does setting your display driver to "raw" (at the bottom of the settings, press Ctrl-p) do anything?
